pkgs/development/interpreters/dyalog/default.nix

```diff
@@ -0,0 +1,65 @@
+# credit to markus1189: https://github.com/markus1189/dyalog-nixos
+{stdenv, xorg, glibc, libiodbc, dpkg, makeWrapper, ncurses5, fetchurl, lib}:
+
+let
+  dyalogLibPath = lib.makeLibraryPath (with xorg; [
+    stdenv.cc.cc.lib
+    glibc
+    libiodbc
+    ncurses5
+  ]);
+in
+stdenv.mkDerivation rec {
+  src = fetchurl {
+    url = "https://www.dyalog.com/uploads/php/download.dyalog.com/download.php?file=${shortVersion}/linux_64_${version}_unicode.x86_64.deb";
+      sha256 = "sha256:03rb9539xd882lk9mwc3m041a6rjdlfihrnfkk9y7rmkfyqrnq2v";
+  };
+
+  pname = "dyalog-apl";
+  version = "18.0.40684";
+
+  shortVersion = lib.concatStringsSep "." (lib.take 2 (lib.splitString "." version));
```

Comment:
Please move this into a let in to not create an env with the value.

Reply:
resolved in https://github.com/NixOS/nixpkgs/pull/137054/commits/dde49e39141bfc381456bef25a1a6c18a50696f2